packages/slice-machine/scripts/start/index.ts

```diff
@@ -47,7 +44,17 @@ async function run(): Promise<void> {
   );
 }
 
-run().catch((err) => {
-  console.error(`[slice-machine] An unexpected error occurred. Exiting...`);
-  console.error("Full error: ", err);
-});
+function start(): Promise<void> {
+  return run().catch((err) => {
+    console.error(`[slice-machine] An unexpected error occurred. Exiting...`);
+    console.error("Full error: ", err);
+  });
+}
+
+export default start;
+
+const isCli = require.main === module;
+
+if (isCli) {
```

Comment:
What's this ?

Reply:
I like being able to call the script directly and/or use it as a model.

So this checks if the script has been required or run directly by `node ../start/index`

Reply:
hmmmm instead of targeting this file you just need to target the BIN file and that's it you don't need this code.

Reply:
fair enough